src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -49,6 +73,16 @@ public void stop() {
         io.setVoltage(0.0, 0.0);
     }
 
+    public void aim() {
+        this.forward = 0;
+        this.rotation =
+                thetaController.calculate(
+                        currentHeading.getAsDouble(), targetHeading.getAsDouble());
+        if (Math.abs(currentHeading.getAsDouble() - targetHeading.getAsDouble()) < 1e-5) {
+            mode = DriveTrainState.MANUAL;
```

Comment:
Let's stay in the AIM state even if the error is small, but supply some boolean that says we are aligned (this will be used by the shooter to determine if we are allowed to shoot)